src/nuclio/common/components/edit-item/edit-item.component.js

```diff
@@ -573,6 +590,14 @@
             return ctrl.selectedClass.id === 'cron';
         }
 
+        /**
+         * Checks if tooltip is visible.
+         * @returns {boolean} `true` only for `secret` and `configMap` classes of volume type, `false` otherwise.
+         */
+        function isTooltipVisible() {
+            return lodash.includes(['secret', 'configMap'], ctrl.selectedClass.id);
```

Comment:
it's better to use classes from the map that you created above. Therefore you will not have duplication of classes